bin/build-all.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-set -e # exiting on error
```

Comment:
Why are you removing this? Don’t we want to fail when a command in the script fails?

Reply:
I had added this earlier. But then I needed the try-catch pattern I add in this PR, so I needed to remove the flag.
One thing we can do is to do it again after the try-catch, so that it fails if any of the later commands fail. I'll make that change now.